off-chain/src/HTTP/Handlers.purs

```diff
@@ -0,0 +1,39 @@
+module Seath.HTTP.Handlers where
+
+import Prelude
+
+import Aeson (encodeAeson, toString)
+import Contract.Prelude (Either(..), log, unwrap)
+import Data.Bifunctor (bimap, rmap)
+import Data.Either (Either)
+import Data.UUID (UUID, genUUID)
+import Effect.Aff (Aff)
+import Effect.Class (liftEffect)
+import Seath.HTTP.Types (IncludeRequest, JSend(..), UID(..), toJsend)
+import Seath.Network.Leader as Leader
+import Seath.Network.Types (IncludeActionError, LeaderNode)
+import Undefined (undefined)
+
+mkHandlers leaderNode = { leader: { includeAction: includeAction leaderNode } }
+
+includeAction
+  :: forall a
+   . Show a
+  => LeaderNode a
+  -> { body :: IncludeRequest a }
+  -- -> Aff (Either String UUID)
+  -> Aff (JSend IncludeActionError UID)
```

Comment:
After some experiments I've decided to use [JSend](https://github.com/omniti-labs/jsend) format to return errors, instead of returning non-200 HTTP code and some error message.

Reply:
Nice! I use that before but didn't know the name.

Reply:
I also didn't know it has a name - figured out while making endpoint =)